go/vt/vtctl/vtctl.go

```diff
@@ -2770,13 +2709,14 @@ func commandValidateSchemaShard(ctx context.Context, wr *wrangler.Wrangler, subF
 	if *excludeTables != "" {
 		excludeTableArray = strings.Split(*excludeTables, ",")
 	}
-	return wr.ValidateSchemaShard(ctx, keyspace, shard, excludeTableArray, *includeViews)
+	return wr.ValidateSchemaShard(ctx, keyspace, shard, excludeTableArray, *includeViews, *includeVSchema)
```

Comment:
The only conflict that was potentially concerning to me was around making sure we had Malcolm's work, and it looks to me like we've got the right changes, so I'm good to go on this.